package.json

```diff
@@ -7,7 +7,8 @@
     "Juan Carlos Farah",
     "Hassan Abdul Ghaffar",
     "Mashkour Abdel Aziz",
-    "Kim Lan Phan Hoang"
+    "Kim Lan Phan Hoang",
```

Comment:
You can remove Mashkour and Hassan, as they were involved in the original template, rather than this app.